cwms-http-client/src/test/java/mil/army/usace/hec/cwms/http/client/TestCwmsHttpLogger.java

```diff
@@ -135,6 +136,51 @@ public void close() throws SecurityException {
         }
     }
 
+    @Test
+    void testCookiesNotLogged() throws IOException {
+        CwmsHttpLoggingInterceptor loggingInterceptor = CwmsHttpLoggingInterceptor.getInstance();
+        loggingInterceptor.setLogLevel(Level.ALL);
+        try(MockWebServer mockWebServer = new MockWebServer()) {
+            String body = readJsonFile("tokens.json");
+            AtomicBoolean foundMessage = new AtomicBoolean(false);
+            Logger.getLogger("okhttp3").addHandler(new Handler() {
+                @Override
+                public void publish(LogRecord record) {
+                    String message = record.getMessage();
+                    if (message.contains("JSESSIONID")) {
```

Comment:
I'm not sure the exact requirements that are making you look for JSESSIONID but I wonder if we shouldn't also be checking that the the value of the id isn't in the logs.

Reply:
For client logs, it is unknown how securely and what format logs are stored so we're making sure session ids, passwords, tokens, etc aren't able to be logged at all. Good point, I should also add a check for the value.